packages/rspack/src/compilation.ts

```diff
@@ -610,6 +611,70 @@ export class Compilation {
 		});
 	}
 
+	get chunks() {
```

Comment:
I would suggest using a similar implementation as we did in `get modules`. https://github.com/web-infra-dev/rspack/blob/8c0f855aa01199ab430350d19a2dd15073868938/packages/rspack/src/compilation.ts#L604-L611

@hardfist any suggestions?

Reply:
@hyf0 The **JsChunk** doesn't have the modules property that's why I've used the **JsStatsChunk**. How could I make this link in a better way?

```javascript
export interface JsChunk {
  files: Array<string>
}
```


```javascript
export interface JsStatsChunk {
  type: string
  files: Array<string>
  id: string
  entry: boolean
  initial: boolean
  names: Array<string>
  size: number
  modules?: Array<JsStatsModule>
  parents?: Array<string>
  children?: Array<string>
  siblings?: Array<string>
}
```